.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Funny thing, you have set the `insert_final_newline` to `true` and this file is missing the `insert_final_newline` 🤭

Reply:
I copy/paste it from @DecampsRenan PR 😅